src/modules/voxelformat/GLTFFormat.h

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file
+ */
+
+#pragma once
+
+#include "MeshExporter.h"
+
+namespace tinygltf {
+	class Model;
+	class Node;
+	class Scene;
+} // namespace tinygltf
+
+namespace voxel {
+
+/**
+ * @brief GL Transmission Format
+ */
+class GLTFFormat : public MeshExporter {
+private:
+	typedef core::DynamicArray<std::pair<int, int>> Stack;
```

Comment:
A small struct that holds these two int values would be better than pulling std::pair into the voxel namespace with a header.